facebookads/__init__.py

```diff
@@ -21,7 +21,7 @@
 from facebookads.session import FacebookSession
 from facebookads.api import FacebookAdsApi
 
-__version__ = '2.10.2'
+__version__ = '2.10.2.1'
```

Comment:
@asiebert can you train @jolancornevin on how to use https://work4labs.atlassian.net/browse/BAT-7765 for package version numbers please?

Reply:
Saw with him a few hours ago, I believe his e143b126fc6ac877c93861a23c6d6f0b8f9c18c7 fixes this. There was some confusion between the SHA1 referenced in requirements and the package version, which is fair enough...

As mentioned then, we might wanna consider hosting our own pip repo for all our custom packages. With an auto-upload when git-pushing, we would make our requirements a lot simpler (and our installs quicker).